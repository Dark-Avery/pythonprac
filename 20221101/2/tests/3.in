a = Triangle((0, 1), (1, 3), (2, 1))
b = Triangle((0, 2), (2, 2), (1, 0))
print(*(f"{n}({bool(x)}):{round(abs(x), 3)}" for n, x in zip("ab", (a, b))))
print(f"{a < b=}, {b < a=}, {a < a=}, {b < b=}")
print(f"{a in b=}, {b in a=}, {a in a=}, {b in b=}")
print(f"{a & b=}, {b & a=}, {a & a=}, {b & b=}")
