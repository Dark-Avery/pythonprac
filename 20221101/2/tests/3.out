a(True):2.0 b(True):2.0
a < b=False, b < a=False, a < a=False, b < b=False
a in b=False, b in a=False, a in a=True, b in b=True
a & b=True, b & a=True, a & a=True, b & b=True
